src/index.js

```diff
@@ -1,29 +1,56 @@
 import puppeteer from 'puppeteer';
+import readline from 'readline';
 import db from './utils/db';
 import ticketManager from './utils/tickets';
 
 const main = async () => {
+  console.log(`Starting service... <q! to exit>`);
+  var should_break = false;
+
+  // create interface for reading user input
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+
   console.log('Launching browser...');
   const browser = await puppeteer.launch();
 
-  console.log('Finding new tickets...');
-  const tickets = await db.getNewTickets();
+  console.log(`Awaiting new tickets...`);
+  // tell program to quit service
+  rl.on(`line`, input => {
+    if (input == `q!`) {
+      should_break = true;
+    }
+  });
 
-  try {
-    // loop through and process tickets
-    for (let ticket of tickets) {
-      await ticketManager.processTicket(browser, ticket);
+  while (true) {
+    if (should_break) {
+      break;
+    }
+    const tickets = await db.getNewTickets();
+    if (tickets.length == 0) {
```

Comment:
nit: `tickets.length > 0` and no else block.

Reply:
Will update